Gemfile

```diff
@@ -39,6 +41,12 @@ group :development do
   gem 'awesome_print', '~> 1.8.0'
 end
 
+group :test do
+  gem 'rspec-rails', '~> 4.0.0'
+  gem 'database_cleaner', '~> 1.8.3'
+  gem 'shoulda', '~> 3.5.0'
+end
+
 # group :test do
```

Comment:
The same group is above

Reply:
You mean I put it together with "group: development,: test"?
In that case I would have to put: development there too

Reply:
Se refiere a que tenes comentado el mismo grupo que tienes en el grupo 44,  osea está repetido pero uno está comentado, puedes borrar el comentado

Reply:
done!